db/schema.rb

```diff
@@ -13,17 +13,20 @@
 
 ActiveRecord::Schema.define(version: 20160815140302) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
   create_table "ahoy_events", force: :cascade do |t|
-    t.uuid     "visit_id",   limit: 16
+    t.integer  "visit_id"
```

Comment:
Yes, indeed we need this.  At first glance, records seem to be creating for ahoy_events and visits